### ISSUE TITLE
State points

### ISSUE THREAD

**Issue by ramess101** [ramess101/Argon_Viscosity#1]
@ibell @ianhbell has requested that I perform Argon viscosity simulations using the Ulrich Deiters potential:

> a nice pair potential function is
> 
> u(r) = {p0/r * exp(p1 * r) + p2 * exp(p3 * r) + p4} / (1 + p5*r^6)
> 
> This can match the ab initio pair potentials of Ar, Kr, Xe very
> well, has a reasonable behaviour in the limit r → 0, and is much
> faster than the "official" ab initio function with Tang-Tönnies
> damping functions. Good values for argon are:
> 
> p0 =  3.139616e+07 K Å
> p1 = -2.817277e+00 Å^-1
> p2 = -2.787276e+03 K
> p3 = -5.836970e-01 Å^-1
> p4 = -3.413785e+02 K
> p5 =  7.371331e-04 Å^-6
> 
> The zero and the minimum are at
> sigma = 3.355134529 Å
> rmin  = 3.755227200 Å
> umin  = -143.4899372 K
> 
> This potential should not only give good virial coefficients, but also
> good viscosities. But note that this is a pure pair potential! For
> simulations at higher densities you have either to add multi-body
> corrections (Richard Sadus is working on this) or to use an effective
> potential. Lennard-Jones is too repulsive at close distances
> (see the attached figure).
> 

The initial state points will be the same as those simulated by Bill:

![image](https://user-images.githubusercontent.com/23408516/47239037-5f7ca600-d3a1-11e8-8337-0d7290d6067f.png)

**Comment by ramess101** [comment]
I will work in real units, with the effective sigma and epsilon values of:

sigma = 0.3355 nm
epsilon = 143.5 K

These do not have to be exactly the same as reported by Ulrich, because we can reduce them back to the dimensionless units afterward. However, this does dictate the real unit state points, i.e., all simulations for T*=1 will be at T = 143.5 K.

As a refresher:

![image](https://user-images.githubusercontent.com/23408516/47239214-e6ca1980-d3a1-11e8-9b6e-c9b615ec710a.png)

**Comment by ibell** [comment]
Sounds good to me!

**Comment by ramess101** [comment]
@ibell 

We have a problem, and I am very surprised that Bill did not run into this as well. Performing simulations along the T*=1 isotherm is very problematic, because this is less than the critical temperature. 

The critical temperature for the LJ fluid is around T*=1.2. Tc for Argon is 150 K, so performing simulations at 143.5 K transitions through an unstable region. For example, I got non-physical (negative) pressures for those densities that are between saturated vapor and liquid. 

Did Bill not have an issue because the WCA doesn't have the attractive nature, so it doesn't have a critical point? At any rate, I should perform simulations at higher temperatures. I don't know what the critical point is for the Uldrich Deiters potential (probably not the same as experiment, based on my experience with true 2-body potentials, it is most likely higher than 150 K. So should I run at 200 K? Or at T*=2, T=287 K? Does it matter to you?

**Comment by ianhbell** [comment]
You're right - the critical point of Lennard-Jones 12-6 is T* = 1.32.  And also right on the WCA(LJ) point - it doesn't have a critical point at all, which is one of the key things that makes the repulsive-only potentials handy.  What do you think about doing simulations both:


a) Along the effective T*=1 line at densities greater than saturated liquid and below saturated vapor densities

b) Along the effective T*=1.5 line, where the fluid is always supercritical?

**Comment by ramess101** [comment]
Yeah we can do both of those options. I will just use the same densities as
Bill for both isotherms unless they are inside the two phase dome.

**Comment by ibell** [comment]
Great - that's precisely what I had in mind.
